torchvision/prototype/transforms/_presets.py

```diff
@@ -26,42 +25,47 @@ def __init__(
         resize_size: int = 256,
         mean: Tuple[float, ...] = (0.485, 0.456, 0.406),
         std: Tuple[float, ...] = (0.229, 0.224, 0.225),
-        interpolation: T.InterpolationMode = T.InterpolationMode.BILINEAR,
+        interpolation: InterpolationMode = InterpolationMode.BILINEAR,
     ) -> None:
         super().__init__()
-        self._resize = T.Resize(resize_size, interpolation=interpolation)
-        self._crop = T.CenterCrop(crop_size)
-        self._normalize = T.Normalize(mean=mean, std=std)
+        self._crop_size = [crop_size]
+        self._size = [resize_size]
+        self._mean = list(mean)
+        self._std = list(std)
+        self._interpolation = interpolation
 
     def forward(self, img: Tensor) -> Tensor:
-        img = self._crop(self._resize(img))
+        img = F.resize(img, self._size, interpolation=self._interpolation)
+        img = F.center_crop(img, self._crop_size)
         if not isinstance(img, Tensor):
             img = F.pil_to_tensor(img)
         img = F.convert_image_dtype(img, torch.float)
-        return self._normalize(img)
+        img = F.normalize(img, mean=self._mean, std=self._std)
+        return img
 
 
 class Kinect400Eval(nn.Module):
     def __init__(
         self,
-        resize_size: Tuple[int, int],
         crop_size: Tuple[int, int],
+        resize_size: Tuple[int, int],
```

Comment:
Unrelated to this PR, but why do we use `Tuple[int, int]` here for the sizes, but plain `int`'s for `ImageNetEval`?

Reply:
This is a good point. This is how the original recipes have been implemented, so I need to do this here too. If you see the resize operates differently if you specify both dimensions vs only 1. Prior merging we should consider adding unions etc here and cleaning up further.